airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -56,42 +53,48 @@
 public class WorkerApp {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(WorkerApp.class);
-  public static final int KUBE_HEARTBEAT_PORT = 9000;
+  public static final int KUBE_HEARTBEAT_PORT = 9876;
```

Comment:
curious, how did you choose this?

Reply:
Just random ports. They aren't exposed as node ports so I don't think it matters?